eth/v1/beacon_state.proto

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 Prysmatic Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+
+package ethereum.eth.v1alpha1;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/any.proto";
+
+// Beacon chain API
+//
+// The beacon chain API can be used to access data relevant to the Ethereum 2.0
+// phase 0 beacon chain.
+service Beacon {
```

Comment:
Beacon sounds off. BeaconChain?

Reply:
Going by the name on https://ethereum.github.io/eth2.0-APIs/#/Beacon, should I change it?